client/trino-client/src/main/java/io/trino/client/uri/TrinoUri.java

```diff
@@ -531,6 +539,21 @@ public boolean isAssumeLiteralUnderscoreInMetadataCallsForNonConformingClients()
         return assumeLiteralUnderscoreInMetadataCallsForNonConformingClients.orElse(false);
     }
 
+    public ZoneId getTimeZone()
+            throws IllegalArgumentException
+    {
+        if (timeZone.isPresent()) {
+            try {
+                return ZoneId.of(timeZone.get());
+            }
+            catch (DateTimeException e) {
+                throw new IllegalArgumentException("Specified timezone is not supported: " + timeZone.get(), e);
```

Comment:
Can the error message give a hint about the format in which timezone needs to be specified? I guess the Java docs are the best reference: https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/ZoneId.html

Should this PR also update the docs?

Reply:
both good points, I'll address them.

Reply:
I'm not sure if it's ok to reference Java docs in error messages, so it's fine as is. Remember to update the docs.